imports/plugins/included/ui-search/lib/components/searchModal.js

```diff
@@ -30,6 +156,7 @@ class SearchModal extends Component {
           textFieldStyle={{ marginBottom: 0 }}
           onChange={this.props.handleChange}
           value={this.props.value}
+          placeholder="enter at least 3 characters"
```

Comment:
Is there a reason why the first letter in the placeholder text is not capitalized?

Reply:
No reason at all
I just believe all should be lowercased

Reply:
Its not even necessary